programming/sr/ruggeduinos/custom_firmware.md

```diff
@@ -249,43 +104,30 @@ To configure a `Robot` object to ignore a Ruggeduino with custom firmware, you w
 The Ruggeduino ID is a 20 character string of mostly numbers, and is output in the robot log when you run a program on your robot with your
 Ruggeduino connected.
 
-After calling `Robot.setup()`, you should call the `ruggeduino_ignore_id`
- method of the robot object, with the ID as an argument.
 You'll need the ID later, so it's best to save it into a variable:
 
 ~~~~~ python
 from sr.robot import *
 
 RUGGEDUINO_ID = "752303138333517171B1" # Replace this with the actual ID
 
-R = Robot.setup()
-
-R.ruggeduino_ignore_id( RUGGEDUINO_ID )
-
-R.init()
-
-R.wait_start()
+R = Robot(ignored_ruggeduinos=["752303138333517171B1"])
```

Comment:
Can we deduplicate the id here? Feels redundant to have it in a constant and then also repeated in a literal.

Reply:
Given that the constant is never used this looks like an error. Having a constant that is then used whenever you want to refer to the board seems like a good example to set.

Reply:
Yes, this is an error. 67e9a3d